### PR TITLE
enchancement: update extract translation to only push on one branch

### PR DESCRIPTION
**Pull request by ga-devfront**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Currently we got multiple PR then we have new translation (possibly with same content), we fix this by pushing only changes in one branche named 'translation-extraction'
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Sponsor company   | @PrestaShopCorp
| How to test?      | Add translations and merge multiple PR on dev and check if we got one or many PR.

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
Can we close this PR @ga-devfront ?

File: .github/workflows/extract-translations.yml
Comment (Quetzacoalt91) [inline .github/workflows/extract-translations.yml:19]: Wrong organization

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
